cluster-scope/base/config.openshift.io/oauths/cluster/oauth.yaml

```diff
@@ -7,3 +7,5 @@ metadata:
     include.release.openshift.io/single-node-developer: "true"
     release.openshift.io/create-only: "true"
   name: cluster
+spec:
+  identityProviders: []
```

Comment:
I don't think we need this change anymore; this file was updated in ee5f117 to have an empty `spec` which should be suffcient for patching.